Unable to Run on Windows 7 x 64

**Description**
Hello, I am new to Wails. Pretty cool project!
I have a problem compiling the app on Windows 7 using ``wails dev`` command.

**Error Output**
```cmd
PS C:\Projects\GO Source Code Files\Web-Applications\Wails\test-app> wails dev
←[0;93mWails CLI←[0m ←[0;31mv2.0.0-beta.27←[0m


←[0;92mExecuting: go mod tidy←[0m
←[0;92mExecuting: wails generate module←[0m
Building application for development...
No Install command. Skipping.
No Build command. Skipping.
Generating bundle assets: Done.
Compiling application: Done.

←[0;92mWatching (sub)/directory: C:\Projects\GO Source Code Files\Web-Applications\Wails\test-app←[0m
←[0;92mUsing Dev Server URL: http://localhost:34115←[0m
←[0;92mUsing reload debounce setting of 100 milliseconds←[0m
INF | Serving assets from disk: C:\Projects\GO Source Code Files\Web-Applications\Wails\test-app\frontend\src
DEB | WebView2 Runtime Version '97.0.1072.55' installed. Minimum version required: 91.0.992.28.
←[0;92mAdded new directory to watcher: C:\Projects\GO Source Code Files\Web-Applications\Wails\test-app\frontend\wailsjs\go←[0m
DEB | [DevWebServer] Serving application at http://localhost:34115
panic: Failed to load shcore.dll: The specified module could not be found.

goroutine 1 [running, locked to thread]:
syscall.(*LazyProc).mustFind(...)
        C:/Program Files/Go/src/syscall/dll_windows.go:323
syscall.(*LazyProc).Call(0xc00011c5d0, {0xc000023760, 0x4, 0x4})
        C:/Program Files/Go/src/syscall/dll_windows.go:339 +0x65
github.com/leaanthony/winc/w32.GetDPIForMonitor(...)
        C:/Users/VH/go/pkg/mod/github.com/leaanthony/winc@v0.0.0-20211202091710-9931d43181ff/w32/shcore.go:15
github.com/leaanthony/winc.(*ControlBase).GetWindowDPI(0xc000222440)
        C:/Users/VH/go/pkg/mod/github.com/leaanthony/winc@v0.0.0-20211202091710-9931d43181ff/controlbase.go:104 +0xd5
github.com/leaanthony/winc.(*ControlBase).scaleWithWindowDPI(0xc000204180, 0x2d0, 0x23a)
        C:/Users/VH/go/pkg/mod/github.com/leaanthony/winc@v0.0.0-20211202091710-9931d43181ff/controlbase.go:466 +0x25
github.com/leaanthony/winc.(*ControlBase).SetSize(0xc000204180, 0x0, 0x0)
        C:/Users/VH/go/pkg/mod/github.com/leaanthony/winc@v0.0.0-20211202091710-9931d43181ff/controlbase.go:184 +0x5a
github.com/wailsapp/wails/v2/internal/frontend/desktop/windows.NewWindow({0x0, 0x0}, 0xc00018a0e0)
        C:/Users/VH/go/pkg/mod/github.com/wailsapp/wails/v2@v2.0.0-beta.27/internal/frontend/desktop/windows/window.go:56 +0x199
github.com/wailsapp/wails/v2/internal/frontend/desktop/windows.(*Frontend).Run(0xc000182630, {0xa173e0, 0xc0001deea0})
        C:/Users/VH/go/pkg/mod/github.com/wailsapp/wails/v2@v2.0.0-beta.27/internal/frontend/desktop/windows/frontend.go:108 +0x78
github.com/wailsapp/wails/v2/internal/frontend/devserver.(*DevWebServer).Run(0xc00018e580, {0xa173e0, 0xc0001deea0})
        C:/Users/VH/go/pkg/mod/github.com/wailsapp/wails/v2@v2.0.0-beta.27/internal/frontend/devserver/devserver.go:139 +0x638
github.com/wailsapp/wails/v2/internal/appng.(*App).Run(0xc00005a320)
        C:/Users/VH/go/pkg/mod/github.com/wailsapp/wails/v2@v2.0.0-beta.27/internal/appng/app_dev.go:47 +0x31
github.com/wailsapp/wails/v2.Run(0xc00003e000)
        C:/Users/VH/go/pkg/mod/github.com/wailsapp/wails/v2@v2.0.0-beta.27/wails.go:24 +0x49
main.main()
        C:/Projects/GO Source Code Files/Web-Applications/Wails/test-app/main.go:26 +0x385
←[0;91mError during retrieving assetdir: Get "http://localhost:34115/wails/assetdir": dial tcp [::1]:34115: connectex: No connection could be made because the target machine actively refused it.←[0m
←[0;91mReloading couldn't be triggered: Please specify -assetdir or -reloaddirs←[0m
```

**Expected behaviour**
App should run normally.

**System Details**
```cmd
PS C:\Projects\GO Source Code Files\Web-Applications\Wails\test-app> wails doctor
←[0;93mWails CLI←[0m ←[0;31mv2.0.0-beta.27←[0m


Scanning system - Please wait (this may take a long time)...Done.

System
------
OS:             Windows 7 Professional
Version:         (Build: 7601)
ID:
Go Version:     go1.17.6
Platform:       windows
Architecture:   amd64

Dependency      Package Name    Status          Version
----------      ------------    ------          -------
WebView2        N/A             Installed       97.0.1072.55
npm             N/A             Installed       6.14.4
*upx            N/A             Available

* - Optional Dependency

Diagnosis
---------
Your system is ready for Wails development!
Optional package(s) installation details:
  - upx : Available at https://upx.github.io/


←[0;93m
If Wails is useful to you or your company, please consider sponsoring the project:
https://github.com/sponsors/leaanthony
←[0m
```

I might have missed something, this is new to me. Please let me know! 
Thank you!

--- a/controlbase.go
+++ b/controlbase.go
@@ -7,6 +7,7 @@ package winc
 
 import (
 	"fmt"
+	"internal/syscall/windows/sysdll"
 	"runtime"
 	"sync"
 	"syscall"
@@ -98,6 +99,11 @@ func (cba *ControlBase) SetHandle(hwnd w32.HWND) {
 	cba.hwnd = hwnd
 }
 
+func (cba *ControlBase) GetSystemDPI() w32.UINT {
+	dpi := w32.GetDpiForSystem()
+	return dpi
+}
+
 func (cba *ControlBase) GetWindowDPI() (w32.UINT, w32.UINT) {
 	monitor := w32.MonitorFromWindow(cba.hwnd, w32.MONITOR_DEFAULTTOPRIMARY)
 	var dpiX, dpiY w32.UINT
@@ -181,8 +187,11 @@ func (cba *ControlBase) clampSize(width, height int) (int, int) {
 func (cba *ControlBase) SetSize(width, height int) {
 	x, y := cba.Pos()
 	width, height = cba.clampSize(width, height)
-	width, height = cba.scaleWithWindowDPI(width, height)
-
+	if sysdll.IsSystemDLL("shcore.dll") {
+		width, height = cba.scaleWithWindowDPI(width, height)
+	} else {
+		width, height = cba.scaleWithSystemDPI(width, height)
+	}
 	w32.MoveWindow(cba.hwnd, x, y, width, height, true)
 }
 
@@ -467,6 +476,17 @@ func (cba *ControlBase) scaleWithWindowDPI(width, height int) (int, int) {
 
 	DPIScaleX := dpix / 96.0
 	DPIScaleY := dpiy / 96.0
+
+	width *= int(DPIScaleX)
+	height *= int(DPIScaleY)
+	return width, height
+}
+
+func (cba *ControlBase) scaleWithSystemDPI(width, height int) (int, int) {
+	dpi := cba.GetSystemDPI()
+
+	DPIScaleX := dpi / 96.0
+	DPIScaleY := dpi / 96.0
 
 	width *= int(DPIScaleX)
 	height *= int(DPIScaleY)

--- a/w32/user32.go
+++ b/w32/user32.go
@@ -125,6 +125,7 @@ var (
 	procMonitorFromRect               = moduser32.NewProc("MonitorFromRect")
 	procMonitorFromWindow             = moduser32.NewProc("MonitorFromWindow")
 	procGetMonitorInfo                = moduser32.NewProc("GetMonitorInfoW")
+	procGetDpiForSystem               = moduser32.NewProc("GetDpiForSystem")
 	procEnumDisplayMonitors           = moduser32.NewProc("EnumDisplayMonitors")
 	procEnumDisplaySettingsEx         = moduser32.NewProc("EnumDisplaySettingsExW")
 	procChangeDisplaySettingsEx       = moduser32.NewProc("ChangeDisplaySettingsExW")
@@ -1093,6 +1094,11 @@ func MonitorFromWindow(hwnd HWND, dwFlags uint32) HMONITOR {
 		uintptr(dwFlags),
 	)
 	return HMONITOR(ret)
+}
+
+func GetDpiForSystem() uint {
+	ret, _, _ := procGetDpiForSystem.Call()
+	return ret
 }
 
 func GetMonitorInfo(hMonitor HMONITOR, lmpi *MONITORINFO) bool {

